app/models/mailboxer/notification.rb

```diff
@@ -162,8 +162,9 @@ def mark_as_deleted(participant)
 
   #Sanitizes the body and subject
   def clean
-    self.subject = sanitize(subject) if subject
-    self.body    = sanitize(body)
+    self.subject   = sanitize(subject) if subject
+    sanitized_body = String.new sanitize(body)
```

Comment:
There is no need for String.new here. That's just causing an unnecessary object creation.